frontend/src/static/js/v_summary.js

```diff
@@ -857,6 +857,7 @@ window.vSummary = {
     this.$root.$on('restoreCommits', (info) => {
       const zoomFilteredUser = this.restoreZoomFiltered(info);
       info.zUser = zoomFilteredUser;
+      info.zFileTypeColors = this.fileTypeColors;
```

Comment:
Just leave a mark here, maybe should shift fileTypeColors to global constant after vuex implemented